packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -70,6 +74,26 @@ const getSizeStyles = (size: TIconButtonProps['size']) => {
         height: ${designTokens.heightForButtonAsBig};
         width: ${designTokens.heightForButtonAsBig};
       `;
+    case '10':
```

Comment:
Since the previous design tokens are now deprecated, I think we should not use them anymore in our code.
I'm referring to the ones above (`heightForButtonAsSmall`, `heightForButtonAsMedium` and `heightForButtonAsBig`).

Reply:
Right. Does anything speak against removing those from the [token definitions](https://github.com/commercetools/ui-kit/blob/fcc083bba5e0c2d3d4e5130172d9bc2bb61423d1/design-system/materials/internals/definition.yaml#L396) as well or could that have side-effects?

Reply:
Addressed [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-92bf6a7b4ce14dccc55ad3c0ee5db75a599f7d62e83a1667d1fbebb0117653b0L60-L71), [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-c10084319127c43b8f3365a94b946414de422e1b5df5272a1c28e43375b35e74L12-L18), [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-c10084319127c43b8f3365a94b946414de422e1b5df5272a1c28e43375b35e74L12-L18) and [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-6fc4fa62a60abfb40503b4213ef3a4acf4799c1f0dd1ab890ad10b1f31970ef5L220-L222). (since no test broke, I removed the now unused tokens from the design-system repo as well). @CarlosCortizasCT

Reply:
Bear in mind this is a breaking change as we export the design tokens and consumers might be using them.
To be honest, I don't expect them to be actually using them, but we can't just remove them without notice. That's a package change that could break consumer's code.

Reply:
@CarlosCortizasCT I just saw that I was smart enough 🤡  to change it the generated design-tokens.ts file and not in the source yaml file. So basically no harm done. Should I still revert it or will it be overwritten anyway?

Reply:
You need to build the packages again so the generated file will be updated and you can commit it with the right data.

Reply:
[Done](https://github.com/commercetools/ui-kit/pull/2830/commits/6ac8da4fda401999d2960ac4d717cf9e2c81ec3a)